pkgs/misc/themes/arc-kde/default.nix

```diff
@@ -0,0 +1,15 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

Comment:
Please add a [`meta` attribute](https://nixos.org/nixpkgs/manual/#chap-meta).

Reply:
Yeah, not really sure what I was thinking.

I was working on this for myself and forgot to add the `meta` stuff before I pushed.